trace/rate_limit.go

```diff
@@ -240,6 +244,10 @@ func (r *rateLimitCollector) Process(pnt akinet.ParsedNetworkTraffic) error {
 			r.NextCollector.Process(pnt)
 			key := requestKey{c.StreamID.String(), c.Seq}
 			r.RequestArrivalTimes[key] = pnt.ObservationTime
+		} else {
+			r.packetCount.Update(client_telemetry.PacketCounts{
+				HTTPRequestsRateLimited: 1,
+			})
```

Comment:
Non-blocking: I think this will work -- the "total" count will be updated.  But I would have more confident in something like this from `PacketCountCollector:

```
		pc.PacketCounts.Update(client_telemetry.PacketCounts{
			Interface:    t.Interface,
			DstHost:      c.Host,
			SrcPort:      t.SrcPort,
			DstPort:      t.DstPort,
			HTTPRequests: 1,
		})
```

So that the per-interface, per-host, and per-port counters are updated.  It does not look like the per-port or per-interface cases look for a missing value -- maybe they should: https://github.com/postmanlabs/postman-insights-agent/blob/fafb76073b8d9b1929789d51d7e78bf81b7ae98a/trace/stats.go#L96

If not the above, please add a comment noting that the breakdowns won't have rate limit?